app/templates/agenda-items/agenda-item.hbs

```diff
@@ -138,8 +138,18 @@
                   @iconClosed="nav-right"
                   @buttonLabel="Stemming"
                 >
-                  <VoteOverview @vote={{this.model.vote}} />
-
+                  {{#if this.model.vote}}
+                    <VoteOverview
+                      @agendaItem={{this.model.agendaItem}}
+                      @vote={{this.model.vote}}
+                    />
+                  {{else}}
+                    <AuAlert
```

Comment:
Can you reuse the logic removed in this PR: https://github.com/lblod/frontend-burgernabije-besluitendatabank/pull/103/commits/4cfa3873af05a6f5128139b108c6697d5be99e79#diff-c2141606cb920557e29cfaa8098d5b210e96f5756444fed2b47e7a216faf0244L144-L152

Reply:
If I use this logic Ember crashes when there is no vote found :/

Reply:
Strange, the condition it based on `this.wasHandled`. I'm taking a look

Reply:
I have not able to reproduce the bug. I commit the change 8acbf6732e264fe0cb8f3e6c824c81f7d51d5fb4